server/package.json

```
@@ -1,4 +1,5 @@
 {
+  "node-version": "12.22.5",
```

A versão 12 já está para perder o suporte faça upgrade para a versão estável atual que é a 16

```suggestion
  "version": "12.22.5",
```

Referência

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#version